install-and-configure/install/provider-installations/openshift-kubecost-install.md

```diff
@@ -60,6 +60,26 @@ If you have not opted to do so during installation, it may be necessary to creat
 
 After installation, wait for all pods to be ready. Kubecost will begin collecting data and may take up to 15 minutes for the UI to reflect the resources in the local cluster.
 
+### Using in-cluster Prometheus
+
```

Comment:
We generally do not recommend users leverage an existing Prometheus because this often leads to numerous downstream issues. I think it would be helpful to add the following warning in this section.

```suggestion

{% hint style="warning" %} 
This installation method is available, but not generally recommended. Please review the following documentation before proceeding. [Documentation](/install-and-configure/advanced-configuration/custom-prom).
{% endhint %}

```
```